kolibri/core/assets/src/views/ExamReport/PageStatus.vue

```diff
@@ -107,6 +141,42 @@
           return this.$tr('notStartedLabel');
         }
       },
+      retakeEnabled() {
+        // The Retake button is enabled only for leaners,
+        // the threshold should be kept in a proper conf and not here (TODO)
```

Comment:
when did you plan on moving this to a conf file? I prefer we don't keep too many TODO's in the code. if there are items we capture that need doing for preventing tech debt, let's make JIRA stories for them